package.json

```diff
@@ -75,7 +75,7 @@
     "lint": "biome check --write && biome lint --write",
     "start": "npm start -w @typeberry/jam --",
     "inspect": "npm run inspect -w @typeberry/jam --",
-    "test": "tsx --test $(find . -type d -name node_modules -prune -type f -o -name '*.test.ts' | tr '\\n' ' ')",
+    "test": "node --test --import tsx $(find . -type d -name node_modules -prune -type f -o -name '*.test.ts' | tr '\\n' ' ')",
```

Comment:
why?

Reply:
I had some issues with lmdb tests but it was resolved on my side. Reverted this, but left the try-catch in tests to properly close the database after tests.